### DISCUSSION TITLE
When do you plan to release 1.0.0 ?

### DISCUSSION THREAD

**Discussion by sean-wallace** [alorence/django-modern-rpc#49] [General]
I don't really want to pin to an alpha but will do it if you think it's going to be a while longer!

**Comment by alorence** [comment]
I understand that the first 1.0 alpha was released too long ago, but I wanted to give enough time to users to test changes and provide feedbacks. The 1.0 release is a big update, with many parts of the code completely rewritten.

The code is now stable in my opinion. I still plan a few updates in the documentation (the [documentation for 1.0 is here](https://django-modern-rpc.readthedocs.io/en/1.0.0a4/)) and the release will be ready in the next weeks. Probably before the end of the year or in the beginning of 2023.

Keep in mind I have no regular contributor to help, and I must work on this project on my free time.

**Comment by sean-wallace** [comment]
Thanks for the update, and the timeline sounds great to me. I really appreciate your work writing and maintaining the package. Merci pour tout le travail que vous y faîtes !

FWIW, we'll probably be deploying our staging apps with 1.0.0a4 to see if we find any lurking bugs.

**Comment by alorence** [comment]
Hi @sean-wallace, the 1.0.0 release has just been published.

Regards
